### ISSUE TITLE
[M1-Dev] Creación de 2 equipos igualados según nivel para jugar un partido

### ISSUE THREAD

**Issue by manujurado1** [manujurado1/SportsBar-IV#54]
Una vez resueltos los problemas en cuanto a inclusión de jugadores y manejo de la disponibilidad al grupo, se plantea resolver el problema que satisfaga la HU #3.
Este problema consiste en separar los jugadores disponibles en 2 equipos igualados según el nivel de sus componentes.

**Comment by manujurado1** [comment]
[ACTUALIZACION] Se considera que un equipo está igualado cuando:
- Los dos mejores jugadores están cada uno en un equipo
- El siguiente jugador con más nivel se asigna al equipo que menos nivel suman sus participantes, siempre que haya hueco en ese equipo (Esto significa que sean menos de la mitad del total de jugadores disponibles).

**Comment by JJ** [comment]
> [ACTUALIZACION] Se considera que un equipo está igualado cuando:
> 
>     * Los dos mejores jugadores están cada uno en un equipo

*un* equipo no puede estar nunca igualado. En todo caso, lo estarán dos.
> 
>     * El siguiente jugador con más nivel se asigna al equipo que menos nivel suman sus participantes, siempre que haya hueco en ese equipo (Esto significa que sean menos de la mitad del total de jugadores disponibles).

Esto es una heurística, que no da en realidad una medida de lo igualados o no que están los dos equipos. Los dos jugadores con más nivel pueden estar a un nivel muy diferentes.

También implica que está claro el concepto de "nivel", un problema que tampoco se ha resuelto.

Es más, estas dos reglas pueden ser contradictorias porque estás hablando de asignar el "siguiente" jugador suponiendo que ya has asignado el resto.

En resumen, tienes que plantear claramente la hu #3, incluyendo la posibilidad de que sea imposible tal partido igualado.

**Comment by manujurado1** [comment]
Cierto, borraré la respuesta [Actualización] ya que no define realmente que dos equipos estén igualados y crearé un nuevo issue que clarifique el concepto de 2 equipos igualados
